pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -90,10 +90,13 @@ def test_min_max_with_nan(self, skipna):
 
     @pytest.mark.parametrize("function", ["min", "max"])
     @pytest.mark.parametrize("skipna", [True, False])
-    def test_min_max_only_nan(self, function, skipna):
+    @pytest.mark.parametrize("box_in_series", [True, False])
```

Comment:
CategoricalIndex, DataFrame col?

Reply:
There are Series tests elsewhere for Categorical min/max, eg the previous PR that touched the Categorical.min/max implementation added the Series tests in reductions/test_reductions.py: https://github.com/pandas-dev/pandas/commit/37526c1ddbe0952dffe16ac0fe158a00aa7b397f

Reply:
@jorisvandenbossche Do you mean do that instead of what I'm doing here, or rather that CategoricalIndex and DataFrame cases should be added to test_reductions.py instead of here?

Reply:
yes the point is to do these in test_reductions.py, adding a similiar missing categorical test there